modules/samples/connectedcup/pom.xml

```diff
@@ -12,32 +12,249 @@
+             by default) and nothing else. We had troubles with other repositories in
+             the past. Therefore configuring additional repositories here should be
+             considered very carefully. -->
+        <repository>
+            <id>wso2-nexus</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>ignore</checksumPolicy>
+            </releases>
+        </repository>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>ignore</checksumPolicy>
+            </releases>
+        </repository>
+        <repository>
+            <id>wso2.snapshots</id>
+            <name>WSO2 Snapshot Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+            </snapshots>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+        </repository>
+        <repository>
+            <id>wso2-staging</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/orgwso2carbonapimgt-174/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>ignore</checksumPolicy>
+            </releases>
+        </repository>
+    </repositories>
+
+
+    <properties>
+		<carbon.kernel.version>4.4.2</carbon.kernel.version>
+		<carbon.device.mgt.version>0.9.2-SNAPSHOT</carbon.device.mgt.version>
+		<paho.mqtt.version>0.4.0</paho.mqtt.version>
+		<smack.wso2.version>3.0.4.wso2v1</smack.wso2.version>
+		<commons-json.version>3.0.0.wso2v1</commons-json.version>
+		<eclipse.osgi.version>3.8.1.v20120830-144521</eclipse.osgi.version>
+		<carbon.iot.device.mgt.version>1.9.2-SNAPSHOT</carbon.iot.device.mgt.version>
+		<javax.ws.rs.version>1.1.1</javax.ws.rs.version>
+		<!--XMPP/MQTT Version-->
+        <smack.wso2.version>3.0.4.wso2v1</smack.wso2.version>
+        <smackx.wso2.version>3.0.4.wso2v1</smackx.wso2.version>
+        <cxf.version>2.6.1</cxf.version>
+        <jackson.version>1.9.0</jackson.version>
+        <commons-httpclient.orbit.version>3.1.0.wso2v2</commons-httpclient.orbit.version>
+        <commons-json.version>3.0.0.wso2v1</commons-json.version>
+        <eclipse.equinox.common.version>3.6.100.v20120522-1841</eclipse.equinox.common.version>
+	</properties>
```

Comment:
Fixed in commit https://github.com/charithag/product-iot-server/commit/463fb50b8fb045fdf7ae25c24e111f879d5153be